.travis.yml

```diff
@@ -8,44 +8,43 @@ env:
   global:
     - COMPOSER_ARGS="--no-interaction"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
-    - TESTS_LAMINAS_CACHE_FILESYSTEM_DIR=/dev/shm
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7
+    - php: 7.3
       env:
         - DEPS=latest
-    - php: 7.1
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.4
       env:
         - DEPS=latest
         - CS_CHECK=true
         - BENCHMARKS=true
         - TEST_COVERAGE=true
-    - php: 7.2
+    - php: 8.0
       env:
+        - COMPOSER_ARGS="$COMPOSER_ARGS --ignore-platform-reqs"
         - DEPS=lowest
-    - php: 7.2
+    - php: 8.0
       env:
+        - COMPOSER_ARGS="$COMPOSER_ARGS --ignore-platform-reqs"
         - DEPS=latest
-    - php: 7.3
+    - php: nightly
       env:
+        - COMPOSER_ARGS="$COMPOSER_ARGS --ignore-platform-reqs"
         - DEPS=lowest
-    - php: 7.3
+    - php: nightly
```

Comment:
Why nightly as well as 8.0?

If we're pinning to `~8.0.0` in the constraint, _nightly_ is future 8.1.0, and thus out-of-scope.

Reply:
Exactly, thats why I've added `allow_failures` for that version. 
This will enable us to early see if we might run out of compatibility in *any* PR rather than waiting until the release.
Its just for visibility tho.